src/ci/docker/run.sh

```diff
@@ -269,14 +269,14 @@ touch $objdir/${SUMMARY_FILE}
 
 extra_env=""
 if [ "$ENABLE_GCC_CODEGEN" = "1" ]; then
-  extra_env="$EXTRA_ENV --env ENABLE_GCC_CODEGEN=1"
+  extra_env="$extra_env --env ENABLE_GCC_CODEGEN=1"
```

Comment:
`extra_env` is set to `""` just before this condition.

Reply:
Yeah, ok, that one doesn't really matter, but each following line was clobbering the ones before.